### PR TITLE
Update BookRepository methods calls

### PR DESCRIPTION
**Pull request by tacionery**
I update the BookRepositoy.method to BookRepository.new.method
According to the https://github.com/hanami/model/blob/master/README.md
The Repository methods are now instance methods instead of class methods

### CODE REVIEW

**Comment by cllns** [comment]
Thanks! This should be merged when `hanami-model` 0.7.0 is released (possibly with `hanami` 0.8.0? #147)

File: source/guides/getting-started.md [hunk 1]
```diff
@@ -538,8 +538,8 @@ describe 'List books' do
   before do
     BookRepository.clear
```
Comment (vasspilka) [inline source/guides/getting-started.md]: You forgot this one ^^

**Comment by cllns** [comment]
Actually these changes were released with [`hanami-model` version `0.6.2`](https://github.com/hanami/model/blob/v0.6.2/lib/hanami/repository.rb) so this should be merged soon, since people are seeing invalid docs.

**Comment by cllns** [comment]
@jodosha @joneslee85 can we get a merge on this as soon as possible? I'm afraid a lot of people are running into this issue. We've had 4 people come to the chat and ask about it in the past couple days. I'm sure many more ran into the issue as well (and didn't join the chat).

I think this change was supposed to be for `0.7.0`but it accidentally got released as `0.6.2`.

**Comment by jodosha** [comment]
@cllns OMG, Thanks for spotting this problem. I'm gonna revert this merge and yank that release.

**Pull request merged**